consensus/src/lib.rs

```diff
@@ -29,7 +29,7 @@ mod state_replication;
 #[cfg(any(test, feature = "fuzzing"))]
 mod test_utils;
 #[cfg(test)]
-mod twins_test;
+mod twins;
```

Comment:
unless twin needs access to consensus internal bits, recommend making it an integration test, see: https://github.com/libra/libra/blob/master/consensus/safety-rules/Cargo.toml#L48

but do try to avoid requiring any feature flags ... I really need to refactor that one to avoid feature flags, sigh.....

Reply:
consensus doesn't have a binary to run like safety-rules.

Reply:
my point was that keeping twins as a unit test is weird :P